### DISCUSSION TITLE
Preload key requests - Next.js Commerce

### DISCUSSION THREAD

**Discussion by FBosler** [vercel/next.js#28452] [Help]
### What version of Next.js are you using?

11.0.0

### What version of Node.js are you using?

Not sure what is using for deployment

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

Lighthouse score points out opportunities to improve page load, most specifically `Preload key requests` as can be seen when running Lighthouse score on `https://demo.vercel.store/`.
This seems to be a result of dynamically loading a component.

### Expected Behavior

Would expect a way to solve this.

### To Reproduce

navigate to  [https://demo.vercel.store/](https://demo.vercel.store/) and run Lighthouse Test

**Comment by brancooo1** [comment]
Thanks for posting this. In case, it couldn't be reproduced, we encounter the same behaviour in our production app - 
Easy FAQ: https://easyfaq.io/

**Comment by rinkusam12** [comment]
any update on this bug?
